edb/server/protocol/protocol.pyx

```diff
@@ -452,6 +463,32 @@ cdef class HttpProtocol:
                     self.server,
                 )
                 return
+            if (path_parts[0] == 'admin' and
+                self.server.is_admin_ui_enabled()
+            ):
+                if path_parts[1:2] == ['protocol']:
+                    database = path_parts[2]
+
+                    out = await binary.eval_buffer(
+                        self.server,
+                        database,
+                        self.current_request.body,
+                    )
+
+                    response.body = out
+                    response.status = http.HTTPStatus.OK
+                    response.content_type = b'application/x.edgedb'
```

Comment:
I'd use something like `application/vnd.edgedb.v019.raw`

Reply:
`vnd` is for officially registered "vendor" protocols. We have to use `x.` before that happens.

Reply:
OK, but I'd still include the version and encoding

Reply:
I did include that..